include/unpack.mk

```diff
@@ -56,7 +56,7 @@ ifeq ($(strip $(UNPACK_CMD)),)
     endif
     # replace zcat with $(ZCAT), because some system don't support it properly
     ifeq ($(PKG_CAT),zcat)
```

Comment:
drive by comment: there's no zcat used here, can that be removed?

Reply:
Some packages use PKG_CAT:=zcat

Reply:
Okay, the hunk comment just irritated me, since there's no $(ZCAT) used to replace anything, it doesn't even exist.

And I just grep'ed a little, the packages setting PKG_CAT are all covered by the auto detection, that can be removed altogether, but that's for another PR.